add: defined relationship link (#10401)
meshery/meshery: .github/ISSUE_TEMPLATE/models.md

1. Prospose the appropriate relationship type, using one of the predefined set of relationship types or suggest a new relationship where an existing type does not fit.
1. Create a Relationship Definition (yaml).
1. Create a policy for evaluation of the relationship (rego).

> 1. Gain insight and understanding by reviewing the provided example of a defined relationship in [Meshery Models](https://github.com/meshery/meshery/pull/9880/files).

Reviewer: ```suggestion
1. Review a prior pull request as an example of how to define a Relationships: https://github.com/meshery/meshery/pull/9880/files
```
